frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.tsx

```diff
@@ -86,7 +90,7 @@ const VirtualHardwareTabFirehose: React.FC<VirtualHardwareTabFirehoseProps> = ({
   const diskWrapper = DiskWrapper.initializeFromSimpleData({
     name: availableCDName,
     type: DiskType.CDROM,
-    bus: DiskBus.VIRTIO,
+    bus: templateValidations.getAValidBus(DiskBus.VIRTIO),
```

Comment:
can we add a method DiskBus.getDefault() and use it in getAValidBus ?

Reply:
done.